apps/krew-ui-angular/tailwind.config.js

```diff
@@ -1,18 +1,14 @@
 module.exports = {
-  prefix: '',
   mode: 'jit',
   purge: {
-    enabled: process.env.TAILWIND_MODE === 'build',
-    content: [
-      './apps/krew-ui-angular/**/*.{html,ts,css,scss,sass,less,styl}',
-      './libs/krew-angular/**/*.{html,ts,css,scss,sass,less,styl}',
-    ],
+    enabled: true,
```

Comment:
Going to always purge.. editor is smart enough to read all the possible classes from tailwind and classes added by themes